app/views/ConferenceList.tsx

```diff
@@ -0,0 +1,25 @@
+import { FlatList, View } from 'react-native'
+
+import styled from 'styled-components'
+import ConferenceCard from '../components/Conference/ConferenceCard'
+import { useAppSelector } from '../hooks'
+
+const Container = styled(View)`
+  flex: 1;
+`
+
+function ConferenceList() {
+  const { conferences } = useAppSelector(state => state.conferences)
+
+  return (
+    <Container>
+      <FlatList
+        contentContainerStyle={{ paddingBottom: 100 }}
```

Comment:
```suggestion
         contentInset={{bottom: 100}}
```
À vérifier si ça marche, mais normalement oui

Reply:
Ne fonctionne pas, je laisse donc tel quel